File: sdk/web/component/sqljson/controller/FuzzySettingController.js
var FuzzySettingController = function FuzzySettingController(cfg)
{
    let ctrl = {};

    ctrl.Save = function (data, onResponse)
    {
        cfg.Connector.Submit(data, "ardvro/component/sqljson/FuzzySetting/Save", function (box)
        {
            onResponse != null ? onResponse(box == null ? null : box.Data) : null;
        });
    };

    ctrl.Delete = function (data, onResponse)
    {
        cfg.Connector.Submit(data, "ardvro/component/sqljson/FuzzySetting/Delete", function (box)
        {
            onResponse != null ? onResponse(box == null ? null : box.Data) : null;
        });
    };

    ctrl.GetAll = function (onResponse)
    {
        cfg.Connector.Submit([], "ardvro/component/sqljson/FuzzySetting/GetAll", function (box)
        {
            onResponse != null ? onResponse(box == null ? null : box.Data) : null;
        });
    };

    ctrl.GetById = function (id, onResponse)
    {
        cfg.Connector.Submit(id, "ardvro/component/sqljson/FuzzySetting/GetById", function (box)
        {
            onResponse != null ? onResponse(box == null ? null : box.Data) : null;
        });
    };

    return ctrl;
};

File: sdk/android/sqljson/controller/FuzzySettingController.java
package ardvro.component.sqljson.controller;

import java.util.ArrayList;

import ardvro.adapter.sso.ssoFuzzy;
import ardvro.core.connection.IConnection;
import ardvro.core.connection.IResponseListener;

public class FuzzySettingController
{
    private IConnection _conection;

    public FuzzySettingController(IConnection connection)
    {
        _conection = connection;
    }

    public void Save(Object data, IResponseListener<ssoFuzzy> onResponse)
    {
        String functionName = "ardvro/component/sqljson/FuzzySetting/Save";
        _conection.Submit(data, functionName, onResponse);
    }

    public void Delete(Object data, IResponseListener<ssoFuzzy> onResponse)
    {
        String functionName = "ardvro/component/sqljson/FuzzySetting/Delete";
        _conection.Submit(data, functionName, onResponse);
    }

    public void GetById(Object id, IResponseListener<ssoFuzzy> onResponse)
    {
        String functionName = "ardvro/component/sqljson/FuzzySetting/GetById";
        _conection.Submit(new Object[] { id }, functionName, onResponse);
    }

    public void GetAll( IResponseListener<ArrayList<ssoFuzzy>> onResponse)
    {
        String functionName = "ardvro/component/sqljson/FuzzySetting/GetAll";
        _conection.Submit(new Object[] { }, functionName, onResponse);
    }

}
